app/views/schedule/show.html.haml

```diff
@@ -0,0 +1,27 @@
+%div{ :data => { 'controller': 'sub-nav', 'sub-nav-displayed-id-value': 'schedule-day-one' } }
+  %section#hero.schedule-hero
+    %nav
+      %ul
+        %li
+          %a#schedule-day-link{ href: "#", displayedId: 'schedule-day-one', :data => { action: 'click->sub-nav#updateDisplay' } }
+            %strong
+              = current_event.conference_day_in_words(1)
+        %li
+          %a#schedule-day-link{ href: "#", displayedId: 'schedule-day-two', :data => { action: 'click->sub-nav#updateDisplay' } }
+            %strong
+              = current_event.conference_day_in_words(2)
+        %li
+          %a#schedule-day-link{ href: "#", displayedId: 'schedule-day-three', :data => { action: 'click->sub-nav#updateDisplay' } }
+            %strong
+              = current_event.conference_day_in_words(3)
+
+  #schedule-container
+    #schedule-day-one{ :data => { 'sub-nav-target': 'hideable' }}
+      - schedule_for_day(@schedule, 1).each do |program_sessions|
+        = render partial: "schedule_block", locals: { program_sessions: program_sessions }
```

Comment:
It is worth noting that there is a shorthand syntax for this: 

```
render 'schedule_block', program_sessions: program_sessions
```

Reply:
✅